crates/core/plugin_sm/tests/plugin.rs

```diff
@@ -227,7 +228,8 @@ mod tests {
     #[serial]
     fn plugin_call_name_and_path() {
         let dummy_plugin_path = get_dummy_plugin_path();
-        let plugin = ExternalPluginCommand::new("test", &dummy_plugin_path);
+        let plugin =
+            ExternalPluginCommand::new("test", &dummy_plugin_path, TEdgeConfigLocation::default());
```

Comment:
For testing purpose, it's better to use `TEdgeConfigLocation::from_custom_root(...)` passing a temporary config containing only the settings requested for the test.

For example see https://github.com/thin-edge/thin-edge.io/blob/main/crates/core/tedge/tests/mqtt.rs#L21